src/keycloak/chart/values.yaml

```diff
@@ -15,6 +15,10 @@ domain: "###ZARF_VAR_DOMAIN###"
 # The primary Keycloak realm
 realm: uds
 
+# UDS Identity Config Environment Variables
+identityConfigEnv:
+  ssoEnabled: false
```

Comment:
nit: this name feels weird. assume this is tied to the specific google idp enabled flag on the realm side? Maybe we can name more specifically?

Reply:
At the moment its specific to google idp, however if we were to add a different idp it would still be required..although if we had two idp's and only wanted one then we'd need separate values so maybe it's better to be specific.